From 7a9176b751d5ace772d7c9f9f6cc1b0bc915b58a Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Sat, 22 Aug 2020 22:51:03 +0200
Subject: [PATCH 36/44] ARM: dts: stm32: Add PDK2 DTO for DH 497-200 with
 ETM0700G0EDH6 display

Add DT overlay for the PDK2 board for DH 497-200 adapter card with
EDT ETM0700G0EDH6 Parallel RGB display attached to it.

Upstream-Status: Inappropriate
Signed-off-by: Marek Vasut <marex@denx.de>
---
 arch/arm/boot/dts/Makefile                    |  1 +
 ...2mp15xx-dhcom-pdk2-overlay-497-200-x12.dts | 24 +++++++++++++++++++
 2 files changed, 25 insertions(+)
 create mode 100644 arch/arm/boot/dts/stm32mp15xx-dhcom-pdk2-overlay-497-200-x12.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 666eff5e3a944..5678f2f0cd750 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1241,6 +1241,7 @@ dtb-$(CONFIG_ARCH_STM32) += \
 	stm32mp157a-stinger96.dtb \
 	stm32mp157c-dhcom-pdk2.dtb \
 	stm32mp15xx-dhcom-pdk2-overlay-460-200-x11.dtbo \
+	stm32mp15xx-dhcom-pdk2-overlay-497-200-x12.dtbo \
 	stm32mp15xx-dhcom-pdk2-overlay-531-100-x21.dtbo \
 	stm32mp15xx-dhcom-pdk2-overlay-531-100-x22.dtbo \
 	stm32mp15xx-dhcom-pdk2-overlay-505-200-x12-ch101olhlwh.dtbo \
diff --git a/arch/arm/boot/dts/stm32mp15xx-dhcom-pdk2-overlay-497-200-x12.dts b/arch/arm/boot/dts/stm32mp15xx-dhcom-pdk2-overlay-497-200-x12.dts
new file mode 100644
index 0000000000000..fc81802831c0c
--- /dev/null
+++ b/arch/arm/boot/dts/stm32mp15xx-dhcom-pdk2-overlay-497-200-x12.dts
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause)
+/*
+ * Copyright (C) 2020 Marek Vasut <marex@denx.de>
+ */
+/dts-v1/;
+/plugin/;
+
+#include "stm32mp15xx-dhcom-overlay-panel-dpi.dtsi"
+
+&display_bl {
+	pwms = <&pwm2 3 500000 PWM_POLARITY_INVERTED>;
+};
+
+&ltdc_dpi_out {
+	remote-endpoint = <&panel_in>;
+};
+
+&panel {
+	compatible = "dataimage,scf0700c48ggu18";
+};
+
+&panel_in {
+	remote-endpoint = <&ltdc_dpi_out>;
+};
-- 
2.42.0

